setup.cfg

```diff
@@ -1,8 +1,61 @@
+[bdist_wheel]
+universal = 1
+
+[metadata]
+name = ansible-bender
+url = https://github.com/ansible-community/ansible-bender
+description = A tool which builds container images using Ansible playbooks
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Tomas Tomecek
+author_email = tomas@tomecek.net
+license = MIT
+license_file = LICENSE
+classifiers =
+    Development Status :: 5 - Production/Stable
```

Comment:
Nit: ini-file indents are usually 2-space.

Reply:
I don't want to reindent this file in this revert PR. That can be done in a separate PR.

Reply:
Although, I'm not convinced that reindenting the file is worthwhile to begin with.

Reply:
Ah, I didn't realize — it doesn't have the usual `git revert` markers.